EIPS/eip-7892.md

```diff
@@ -72,24 +84,75 @@ To facilitate these changes on the execution layer, the `blobSchedule` object sp
 }
 ```
 
-On the consensus layer, a new parameter is added to the configuration:
+### Consensus layer configuration
 
-```
-BLOB_SCHEDULE:
-  - EPOCH: 348618
+A new `BLOB_PARAMETER_SCHEDULE` field is added to consensus layer configuration, containing a sequence of entries representing blob parameter changes after `ELECTRA_FORK_EPOCH`. This replaces the deprecated `MAX_BLOB_PER_BLOCK{_FORKNAME}` convention for future forks, while preserving existing entries for previous forks.
+
+Entry rules:
+
+- One entry per fork that changes blob parameters (regular or BPO forks).
+- `EPOCH` and `MAX_BLOBS_PER_BLOCK` required for all entries.
+- `FORK_VERSION` required only for BPO forks (regular forks specify this in source).
```

Comment:
Personally, I'm not a fan of `FORK_VERSION` being optional. It just feels error prone and a little confusing. I would rather it be listed twice, then deduplicated in `compute_fork_version`.

For example:

```
GLOAS_FORK_EPOCH: 440000
GLOAS_FORK_VERSION: 0x0B000000

BLOB_SCHEDULE:
  ...
  - EPOCH: 440000  ## GLOAS_FORK_EPOCH
    MAX_BLOBS_PER_BLOCK: 72
    FORK_VERSION: 0x0B000000  ## GLOAS_FORK_VERSION

Reply:
i know it breaks old deployments but imo it shouldn't be optional in the spec at the very least

Reply:
I also prefer verbosity, but expected the duplication to not be welcome. Can certainly add it.